connectivity/check/policy.go

```diff
@@ -138,6 +139,20 @@ func deleteCNP(ctx context.Context, client *k8s.Client, cnp *ciliumv2.CiliumNetw
 	return nil
 }
 
+var (
+	defaultDropReason = func(flow *flowpb.Flow) bool {
+		return flow.GetDropReasonDesc() != flowpb.DropReason_DROP_REASON_UNKNOWN
+	}
+
+	policyDenyReason = func(flow *flowpb.Flow) bool {
+		return flow.GetDropReasonDesc() == flowpb.DropReason_POLICY_DENY
+	}
+
+	defaultDenyReason = func(flow *flowpb.Flow) bool {
+		return flow.GetDropReasonDesc() == flowpb.DropReason_POLICY_DENIED
+	}
+)
```

Comment:
Any reason to not just make these plain `func`s?

Reply:
Good catch :100: